src/com/devsmart/plotter/ElementLineRenderer.java

```diff
@@ -0,0 +1,91 @@
+package com.devsmart.plotter;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+
+import java.util.List;
+
+public final class ElementLineRenderer implements DataRenderer {
+    public static class XYPair {
+        public float x;
+        public float y;
+        public String atomicSymbol;
+
+        public XYPair(float x, float y, String atomicSymbol) {
+            this.x = x;
+            this.y = y;
+            this.atomicSymbol = atomicSymbol;
+        }
+    }
+
+    private final List<XYPair> mLinesFromOriginList;
+    private final Paint mPaint = new Paint();
+    private int mColor;
+    private int mYPosition;
+    private String mEmissionType;
```

Comment:
This would be the first direct reference to SciAps logic.

This library has traditionally just been a generic graphing library.

Reply:
updated to LineWithType Renderer